internal/kubernetes/clients/clients.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 
 	v1alpha1approverpolicy "github.com/cert-manager/approver-policy/pkg/apis/policy/v1alpha1"
-	v1certmanager "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
+	certmanagerv1 "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
Nit: `cmapi`

Reply:
Addressed in 71eab72d09063f2ef5c7251948d38d7cee9a658b